### PR TITLE
QuestDB 8.0

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Replaces STRING column type with the new VARCHAR type. Also, changes the type of a few boolean flag columns from SHORT to BYTE.

### CODE REVIEW

**Comment by puzpuzpuz** [comment]
Hi @nickitat 

Could you take a look at this PR?

File: questdb-partitioned/create.sql [hunk 1]
```diff
@@ -1,73 +1,73 @@
 CREATE TABLE hits
 (
     WatchID long,
-    JavaEnable short,
-    Title string,
-    GoodEvent short,
+    JavaEnable byte,
```
Comment (nickitat) [inline questdb-partitioned/create.sql:4]: sorry for an obvious question, but is `byte` big enough to represent all the possible values?

Comment (puzpuzpuz) [reply questdb-partitioned/create.sql:4]: Yes, this and many other columns are boolean-like, i.e. they take only 0 and 1 as their values. I've checked distinct value for all columns before changing their type.

**Comment by puzpuzpuz** [comment]
@nickitat many thanks for the review!

**Pull request merged**